python/cudf/cudf/_lib/aggregation.pyx

```diff
@@ -163,6 +163,7 @@ cdef class Aggregation:
         cdef Aggregation agg = cls()
         agg.c_obj = move(libcudf_aggregation.make_product_aggregation())
         return agg
+    prod = product
```

Comment:
Is this necessary for pandas compatibility? If not, we probably don't need to create the unnecessary alias.

Reply:
This may not be needed for pandas compatibility. But `groupby.prod` is necessary.